config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits.yaml

```diff
@@ -104,6 +104,29 @@ presubmits:
     annotations:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api
       testgrid-tab-name: pr-integration
+  - name: pull-cluster-api-e2e
```

Comment:
Should we also add a periodic (and possibly also a postsubmit) job for this as well?

Reply:
this e2e tests that machine CRD and machine webhook is validating correctly -- I don't think we'd need a post submit or periodic for this. why do you think we need those?

Reply:
I'd like to have it as a signal that the current state of HEAD (or a particular release branch) is in a good state for release signal.

Reply:
sure, no problem, one sec